regression-tests/sparktkregtests/testcases/frames/frame_timeseries_test.py

```diff
@@ -30,9 +30,20 @@
 
 #Change filename to the location of the csv file used for the test
 filename = "../datasets/timeseriesstats.csv"
+goldfile = "../../../datasets/dickeyfuller.txt"
```

Comment:
Why is it that we need a gold file for dickey-fuller, whereas with durbin-watson you're able to just call statsmodel from the test and compare the results?